libraries/AP_HAL/board/linux.h

```diff
@@ -227,6 +227,7 @@
 #define HAL_BARO_DEFAULT HAL_BARO_MS5611_SPI
 #define HAL_BARO_MS5611_NAME "ms5611"
 #define HAL_COMPASS_DEFAULT HAL_COMPASS_BBBMINI
+#define HAL_AIRSPEED_MS4515DO_I2C_BUS 2
```

Comment:
@juvinski Yes, the macro is defined in the driver, but not used anywhere in the code.

@tridge Yes, we certainly do. It's no so easy in Linux. We probably also need to know if the bus has external access or not. I'm more and more convinced that a macro with a bitmask in each board definition (in AP_HAL/boards) is the easiest way to go for now.